### ISSUE TITLE
Editing 

### ISSUE THREAD

**Issue by i3visio** [sandstorm-io/sandstorm#2294]
At least under a dev installation, it is not possible to edit the display name of the identities created linked to an account. I have tried to create some aliases for the Admin account and I can modify the rest of the variables (photo, alias, name) but not the Display Name as show in the GUI.

**Comment by dwrensha** [comment]
Hi!

Can you be more specific about how you are trying to change the display name? It seems to work for me:

![alice1](https://cloud.githubusercontent.com/assets/495768/17271082/b378231e-5640-11e6-932f-be4da43ca34f.png)
![alice2](https://cloud.githubusercontent.com/assets/495768/17271083/b603efaa-5640-11e6-8b0c-fa21feecb864.png)

**Comment by i3visio** [comment]
Yeah, I mean the name below the "big one". In your screenshot, it's here:
![name_change](https://cloud.githubusercontent.com/assets/8018610/17271346/684139ba-5679-11e6-803e-1cc84acb2cea.png)

**Comment by dwrensha** [comment]
Oh, right! We call that the "intrinsic name". It cannot be edited because it's the unique identifier for the dev account. Note that for Github accounts, it's the github username that goes there, for email accounts, it's the email address, etc.

**Comment by i3visio** [comment]
Okey, I got it. We have three identifiers including Full Name, Intrinsic Name and Handle, right? I feel that we should close the issue then. 

Anyway, just for me to know, if i have created some identities for that Admin account, can they be deleted somehow? I was trying to create a new identity with a different "intrinsic name" and it is not possible for me...

**Comment by dwrensha** [comment]
> We have three identifiers including Full Name, Intrinsic Name and Handle, right? 

Right.

> Anyway, just for me to know, if i have created some identities for that Admin account, can they be deleted somehow?

You should be able to unlink an identity from the account by clicking the "unlink this identity" button in the profile editor.

> I was trying to create a new identity with a different "intrinsic name" and it is not possible for me...

It should be possible to create a dev identity with any intrinsic name by using the "custom" button in the dev login form. The value you enter into that field will be the intrinsic name.

**Comment by dwrensha** [comment]
Hm. Maybe we should add some hovertext over the intrinsic name to provide a fuller explanation of what it means.

**Comment by i3visio** [comment]
Yeah, it should be enough to keep the user advised of what it means and that it cannot be changed or deleted later. Thanks for the effort.
